modules/ROOT/pages/sync/auth.adoc

```diff
@@ -1,13 +1,24 @@
 include::{partialsdir}/attributes.adoc[]
 
-= Binding the {sync-service} service to a Keycloak instance
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: sync_auth
 
-This section will help you to bind a Keycloak service to the {sync-service} service.
+[id='{context}_authentication']
+= Authentication
+
+The {sync-service} by default do not provide any any authentication and authorization mechanism.
```

Comment:
@psturc Actually tried to execute this steps and see how long this takes. It's not simple or convinient which made me question entire instruction. pinged @ziccardi to see if we can just get config directly from keycloak server and paste that directly as content to environment variable. If this is working we can document that - possibly instruction will change to 3 simple steps.

Reply:
This is going to be separate ticket as @ziccardi is out sick.